torch/_inductor/lowering.py

```diff
@@ -3383,6 +3389,7 @@ def backend_reduce_str(reduce):
             zero_out,
         )
         buffer.name = V.graph.register_buffer(buffer)
+        V.graph.register_operation(buffer)
```

Comment:
also confused by this one? Did you mean `register_operation(zero_out)`?

Reply:
Nope, `ComputedBuffer` is an `OperationBuffer` here.